src/graphdb.js

```diff
@@ -193,34 +256,65 @@ function graphDBInferenceSchema (graphbSchema, addMutations) {
     // query
     r += `type Query {\n`;
     gdbs.nodeStructures.forEach(node => {
-        let nodeCase = toPascalCase(node.label); 
+        let nodeCase = replaceCleanseLabel(node.label);
+        nodeCase = toPascalCase(nodeCase);
         r += `\tgetNode${nodeCase}(filter: ${nodeCase}Input): ${nodeCase}\n`;
         r += `\tgetNode${nodeCase}s(filter: ${nodeCase}Input, options: Options): [${nodeCase}]\n`;
     });
     r += '}\n\n';
 
     // mutation
-    if (addMutations) {        
+    if (addMutations) {
         r += `type Mutation {\n`;
         gdbs.nodeStructures.forEach(node => {
-            let nodeCase = toPascalCase(node.label);         
-            r += `\tcreateNode${nodeCase}(input: ${nodeCase}Input!): ${nodeCase}\n`;
-            r += `\tupdateNode${nodeCase}(input: ${nodeCase}Input!): ${nodeCase}\n`;
-            r += `\tdeleteNode${nodeCase}(_id: ID!): Boolean\n`;
+            let nodeCase = replaceCleanseLabel(node.label);
+            let invalidNode = node.label !== nodeCase ? true : false;
+            nodeCase = toPascalCase(nodeCase);
+            if (invalidNode) {
+                r += `\tcreateNode${nodeCase}(input: ${nodeCase}Input!): ${nodeCase} @alias(property: "createNode${node.label}")\n`;
```

Comment:
I don't think these aliases are used for mutations. The APIs seem to work without it (I tried commenting out this alias logic and then creating an ec2 node and it still works).